modules/vsphere-problem-detector-operator.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-operator_{context}"]
+= vSphere Problem Detector Operator
+
+[discrete]
+== Purpose
+
+The {operator-name} checks the cluster for the most common installation and misconfiguration issues.
+
+[discrete]
+== Project
+
+link:https://github.com/openshift/vsphere-problem-detector[vsphere-problem-detector]
+
+[discrete]
+== Configuration
+
+No configuration is required.
+
+[discrete]
+== Notes
+
+* The {operator-name} supports installations on vSphere and {rh-openstack-first}.
+* The {operator-name} uses the `vsphere-cloud-credentials` to communicate with vSphere.
```

Comment:
Please confirm or correct me on the credentials.  If this information is absolutely not needed by customers, please let me know that too.